images/ceph/Makefile

```diff
@@ -124,15 +124,15 @@ csv: $(OPERATOR_SDK) $(YQ) ## Generate a CSV file for OLM.
 csv-clean: $(OPERATOR_SDK) $(YQ) ## Remove existing OLM files.
 	@rm -fr ../../cluster/olm/ceph/deploy/* ../../cluster/olm/ceph/templates/*
 
-# list-image creates list of images for offline installation
-list-image:
-	@echo "producing list of images for offline installation";\
-	# remove the file if already exists
-	rm -f $(MANIFESTS_DIR)/images.txt;\
+# reading from a file and outputting to the same file can have undefined results, so use this intermediate
+IMAGE_TMP="/tmp/rook-ceph-image-list"
+list-image: ## Create a list of images for offline installation
```

Comment:
`make help` can generate help text only if the target is followed by a double `#` and then the description, so I make that change here.